test_runner/src/main/kotlin/ftl/presentation/cli/firebase/test/android/locales/AndroidLocalesListCommand.kt

```diff
@@ -26,6 +30,14 @@ class AndroidLocalesListCommand :
     )
     override var configPath: String = FtlConstants.defaultAndroidConfig
 
+    override val out = outputLogger {
+        @Suppress("UNCHECKED_CAST")
+        when (this) {
+            is List<*> -> (this as List<Locale>).toCliTable()
```

Comment:
IMO `toCliTable` should be moved to presentation layer

Reply:
Isn't it already there?

Reply:
ah, nvm :)

Reply:
@piotradamczyk5 it will be auto moved when another ticket is merged